### ISSUE TITLE
[Docs] Seperate files for CSS of different sections

### ISSUE THREAD

**Issue by Anshika-Jain02** [OSCode-Community/OSCodeCommunitySite#447]
### Current State:
*Only one major CSS file containing all code for all sections. This makes code non-modular and difficult to maintain and to make changes to it.*

### Desired State:
*Different CSS files for each section under different folders by section name to make the overall code more modular and maintainable.*

**Please assign me this issue under Gssoc'23 @harshsingh32**

---

**Comment by akshathar-navada** [comment]
Yes , pls go ahead @Anshika-Jain02 . Pull the repo properly before making required changes and then open a PR. We will review and would be happy to merge your PR. Make sure that none of the applied styles will change :)

**Comment by Yashagrahari07** [comment]
I will separate files for CSS of different sections in different CSS files. Please assign it to me.
